Rationale.md

```diff
@@ -1,5 +1,17 @@
 # Design Rationale
 
+This document describes rationales for WebAssembly's design decisions, acting as
+footnotes to the main design text, keeping the main specification easier to
+read, and making it easier to revisit decisions later without having to plow
+through all the issues and pull requests. This rationale document tries to list
+how decisions were made, and where tradeoffs were made for the sake of language
+ergonomics, portability, performance, security, and Getting Things Done.
+
+WebAssembly was designed incrementally, with multiple implementations being
+pursued concurrently. Design desisions are made with a solid rationale, but as
```

Comment:
1. "decisions" has a `c` in it.
2. purely a style suggestion, feel free to ignore, but I think that this would be nicer: "Design decisions are made with  **what we believe is** a solid rationale"